torch/csrc/jit/register_c10_ops.cpp

```diff
@@ -144,7 +144,7 @@ Operator createOperatorFromC10(const c10::OperatorHandle& op) {
               // in the list are constants
               const std::vector<double>& value = iter->toDoubleListRef();
               std::vector<Value*> info(value.size());
-              for (int value_index = 0; value_index < value.size(); ++value_index) {
+              for (unsigned value_index = 0; value_index < value.size(); ++value_index) {
```

Comment:
shouldn't this be `size_t` to match the type of value.size()

Reply:
🤔 That would make sense.